Defs/TraderKindDefs/TraderKinds_Miner.xml

```diff
@@ -25,7 +25,7 @@
         <thingDef>ComponentSpacer</thingDef>
         <countRange>
           <min>10</min>
-          <max>20</max>
+          <max>200</max>
```

Comment:
This breaks the balance of the game. We cannot increase the maximum advanced component amount to 200